code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -210,3 +221,30 @@
 		/obj/item/weapon/FixOVein = 1,
 		/obj/item/stack/medical/advanced/bruise_pack = 1,
 	)
+
+/obj/item/weapon/storage/fancy/tray/update_icon()
+	cut_overlays()
+
+	var/list/types_and_overlays = list(
+	/obj/item/weapon/bonesetter = "tray_bonesetter",
```

Comment:
would you mind indenting these and putting the closing parenthesis on a new line?

Reply:
ok. it is done